### PR TITLE
Feature/testing2

### PR DESCRIPTION
**Pull request by sydnerd**
### Acceptance Criteria
As a dev, there is testing
### Issues Resolved
Tested add dog and add event views
### Code Reviewers
@alexmkio

### CODE REVIEW

File: cypress/integration/add_dog_view.js [hunk 1]
```diff
@@ -1,11 +1,52 @@
+import { aliasQuery } from "../utils/graphql-test-utils";
```
Comment (alexmkio) [inline cypress/integration/add_dog_view.js:1]: I think you can delete this

File: cypress/utils/graphql-test-utils.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import mockData from '../fixtures/user_query.json'
```
Comment (alexmkio) [inline cypress/utils/graphql-test-utils.js:1]: I think you can delete this file

File: src/components/header/Header.js [hunk 3]
```diff
@@ -8,10 +8,10 @@ export const Header = () => {
         <h1 className='title'>Pawty Trainer</h1>
       </section>
       <nav>
-        <Link to ='/'>Dashboard</Link>
-        <Link to ='/create'>Add Dog</Link>
-        <Link to ='/add_event'>Add Event</Link>
-        <Link to ='/calendar'>Calendar</Link>
+        <Link to ='/'className='dashboard-nav'>Dashboard</Link>
```
[the remark this answers is not in the document]
Comment (sydnerd) [reply src/components/header/Header.js:11]: yes

**Review by alexmkio — APPROVED** [review]

File: cypress/integration/add_dog_view.js [hunk 4]
```diff
@@ -1,11 +1,50 @@
...
+  it('Should have the header with a title, and 4 links', () => {
+    cy.get('.title').should('include.text', 'Pawty Trainer')
+      .get('.dashboard-nav').should('include.text', 'Dashboard')
+      .get('.add-dog-nav').should('include.text', 'Add Dog')
+      .get('.add-event-nav').should('include.text', 'Add Event')
+      .get('.calendar-nav').should('include.text', 'Calendar')
   });
 
+  it('Should have a form with a title, 3 inputs, and a button', () => {
+    cy.get('.train-dog-title').should('include.text', 'Train your dog!')
```
Comment (alexmkio) [inline cypress/integration/add_dog_view.js:18]: This one is failing because:
 + expected - actual

      -'Train Your Dog!'
      +'Train your dog!'

**Review by alexmkio — CHANGES_REQUESTED** [review]

**Pull request merged**